libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/task/RemoveTilesTask.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.route.onboard.task
+
+import android.os.AsyncTask
+import com.mapbox.geojson.Point
+import com.mapbox.navigation.route.onboard.OnOfflineTilesRemovedCallback
+import com.mapbox.navigator.Navigator
+
+internal class RemoveTilesTask(
```

Comment:
> Class "RemoveTilesTask" is never used

Reply:
it's definitely should be here, we have to provide opportunity to remove tasks